src/test/resources/cppcheck/self_closing_scope_limited.xml

```diff
@@ -4,13 +4,3 @@
     <errors>
     </errors>
 </results>
```

Comment:
@uhafner 
I want to support only parsing content of files. Not parsing a build log file.

Mixing content, like in a build log file, introduces problems that I don't want to solve. I think it is a fair requirement to put on the users, to supply valid XML.

With this change, the CPPCheck output has to be a valid XML. This file will not be parsable anymore with this PR:
https://github.com/tomasbjerre/violations-lib/blob/1.130/src/test/resources/cppcheck/self_closing_scope_limited.xml

What do you think? How will that affect your plugins?

Reply:
I think it makes sense to support only valid XML files. I have this restriction on many other parsers as well. So I don't think that this will break anything.